src/main/java/uk/gov/hmcts/reform/sendletter/services/ftp/FtpClient.java

```diff
@@ -80,14 +80,14 @@ public void upload(FileToSend file, String serviceFolder, SFTPClient sftpClient)
 
         retryTemplate.execute(arg -> {
             try {
-                logger.info(
```

Comment:
The default level of app insights is info and higher so I downgraded these instead of removing as they could still be useful but happy to delete as well